src/pipecat/processors/frameworks/rtvi.py

```diff
@@ -1437,7 +1477,7 @@ async def process_frame(self, frame: Frame, direction: FrameDirection):
         elif isinstance(frame, RTVIActionFrame):
             await self._action_queue.put(frame)
         elif isinstance(frame, LLMConfigureOutputFrame):
```

Comment:
Here we rely on `LLMConfigureOutputFrame` to determine if the pipeline is skipping TTS, but elsewhere (the assistant aggregator, I believe?) we were just inspecting text frames as they were coming in and determining on the fly, based on their `skip_tts` values, whether TTS was being skipped. Why the different approaches?